### PR TITLE
Endpoints

### PR DESCRIPTION
**Pull request by scholtz-gnome**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,7 +6,7 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
-output.txt
+endpoints.md
```
Comment (jcqvisser) [inline .gitignore:9]: What's going on here?

**Review by jcqvisser — APPROVED** [review]
LGTM

File: .gitignore (hunk 1 above)
Comment (scholtz-gnome) [reply .gitignore:9]: I had a random output.txt file I was using a while back and didn't want it getting in the way. endpoints.md is a local markdown file where I've listed my routes. Is that something I might want to include under docs?

Comment (jcqvisser) [reply .gitignore:9]: It would be useful as a precursor to API docs (OpenApi, Swagger, Postman or whatever)

**Comment by scholtz-gnome** [comment]
Rather going to merge into dev.

**Pull request closed without merging**